pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -104,7 +104,7 @@ struct bpf_map_def SEC("maps/port_bindings") port_bindings = {
  */
 struct bpf_map_def SEC("maps/udp_port_bindings") udp_port_bindings = {
     .type = BPF_MAP_TYPE_HASH,
-    .key_size = sizeof(__u16),
+    .key_size = sizeof(port_binding_t),
```

Comment:
Do we need the key to port_binding_t here? Since netns is always zero, we should be able to continue using u16

Reply:
`net_ns` is not zero for tcp connections, only udp.

Reply:
The comment is out of date, let me update it.

Reply:
This map is only for udp port bindings though. But I made this comment, and then realized that the populatePortMappings depends on the tcp/udp port binding maps being the same, so never mind